src/Elliptic/Systems/Poisson/FirstOrderSystem.hpp

```diff
@@ -33,52 +33,77 @@ namespace Poisson {
 /*!
  * \brief The Poisson equation formulated as a set of coupled first-order PDEs.
  *
- * \details This system formulates the Poisson equation \f$-\Delta u(x) =
- * f(x)\f$ as the set of coupled first-order PDEs
+ * \details This system formulates the Poisson equation \f$-\Delta_\gamma u(x) =
```

Comment:
Just a nitpick, but I prefer \nabla^2 to \Delta for the Laplacian operator

Reply:
It's `\Delta` in a lot of places in the documentation already and I don't see a strong reason to change that, in particular because also much of the (NR) literature uses `\Delta`.

Reply:
I don't feel strongly about it...I'm ok leaving it as is.